### ISSUE TITLE
SesameDriver 

### ISSUE THREAD

**Issue by skodapetr** [openlink/virtuoso-opensource#279]
It looks like Virtuoso sesame driver does not properly support Dataset, code sample:

```
public static void run(RepositoryConnection conn, ValueFactory valueFactory) throws Exception {
    final URI source = valueFactory.createURI("http://source");
    final URI target = valueFactory.createURI("http://target");
    // Prepare repositories.
    conn.clear(source, target);
    conn.add(valueFactory.createURI("http://localhost/resource/1"), 
            valueFactory.createURI("http://localhost/ontology/value"), 
            valueFactory.createLiteral("Hi!"), source);
    // Execute update query.
    final Update update = conn.prepareUpdate(QueryLanguage.SPARQL, "INSERT { ?s <http://predicate> ?o } WHERE { ?s ?p ?o }");
    final DatasetImpl dataset = new DatasetImpl();            
    dataset.addDefaultGraph(source);
    // dataset.addDefaultGraph(target); // Only one default graph can be specified.
    dataset.setDefaultInsertGraph(target);            
    update.setDataset(dataset);
    update.execute();
}    
```

Before update query state:

```
Triple list for: http://source
    (http://localhost/resource/1,http://localhost/ontology/value,Hi!)
Triple list for: http://target
```

After update has been executed:

```
Triple list for: http://source
    (http://localhost/resource/1,http://localhost/ontology/value,Hi!)
    (http://localhost/resource/1,http://predicate,Hi!)
Triple list for: http://target
```

While the result should be:

```
Triple list for: http://source
    (http://localhost/resource/1,http://localhost/ontology/value,Hi!)
Triple list for: http://target
    (http://localhost/resource/1,http://predicate,Hi!)
```

**Comment by HughWilliams** [comment]
@skodape: We are looking into this ...

**Comment by smalinin** [comment]
Fixed in Sesame Provider build 1.20
